src/app/account/mfa/sms/panel.tsx

```diff
@@ -81,10 +87,18 @@ export function ManageSMSMFAPanel() {
             setIsVerifying(true)
         } catch (error) {
             const errorMessage = errorToString(error)
-            notifications.show({
-                message: errorMessage,
-                color: 'red',
-            })
+
+            if (errorMessage?.includes('`phone_number` must be a `phone_number`')) {
+                notifications.show({
+                    message: 'Please enter a valid phone number.',
+                    color: 'red',
+                })
+            } else {
+                notifications.show({
+                    message: errorMessage,
+                    color: 'red',
+                })
+            }
```

Comment:
I know you didn't start this, but seems like we should probably add the error to the form so it displays inline vs a notification